src/appleseedmaya/exporters/shadingnodeexporter.cpp

```diff
@@ -541,7 +541,7 @@ void ShadingNodeExporter::exportValue(
     {
         MVector value;
         if (AttributeUtils::get(plug, value))
-            ss << "vector " << value.z << " " << value.y << " " << value.z;
+            ss << "vector " << value.x << " " << value.y << " " << value.z;
```

Comment:
Wow, sucks. The kind of typo that code reviews are supposed to catch. Somehow we missed this one :)

I can imagine it causing a ton of subtle and not-so-subtle problems indeed. Since when is it in?

Reply:
i already merged. I was going nuts trying to figure out why my eta+k values from f0/f90 were strange and copper was green. After reviewing the AFF paper, conductor fresnel, coated fresnels, etc.. it was that single character typo. It truly sucks, but alas it's no more.